prep-html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Skeleton Exercise </title>
```

Comment:
Hi Jonathan, be sure to close your `<head>` tag before opening your `<body>` tag.

Reply:
[
![Skeleton Screenshot](https://user-images.githubusercontent.com/96153171/148865015-1f1f4e0e-9ca8-40fe-8c98-d5f0ee045d57.png)
](url)

Closed head tag before opening body tag